regress/pesterTests/SFTP.Tests.ps1

```diff
@@ -208,7 +208,7 @@ Describe "SFTP Testcases" -Tags "CI" {
 
            #validate file content.
            $($ExpectedOutput).split($expectedOutputDelimiter) | foreach {
-              $outputFilePath | Should Contain ([RegEx]::Escape($_))
+              Test-Path ([RegEx]::Escape($_))
```

Comment:
There is no should here. This line just return $true or false. It does not validate the file exists